test_weather_client.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
looks like redundant comments from some ide or notebook platform, should be removed.